frontend/src/rtk/features/systemUserApi.ts

```diff
@@ -0,0 +1,78 @@
+import { api } from './api';
+import { SystemRight, SystemUser, VendorSystem } from '@/types';
+
+const mockRightsActionsArray = [
+  { name: 'Lese', on: true },
+  { name: 'Skrive', on: false },
+  { name: 'Signere', on: true },
+  { name: 'Les arkiv', on: false },
+  { name: 'Launch-Rune´s-Rocket', on: true },
+];
+
+interface CreationRequest {
+  integrationTitle: string;
+  selectedSystemType: string;
+}
+
+const apiWithTag = api.enhanceEndpoints({ addTagTypes: ['SystemUsers'] });
+
+export const systemUserApi = apiWithTag.injectEndpoints({
+  endpoints: (builder) => ({
+    getSystemUsers: builder.query<SystemUser[], void>({
+      query: () => `systemuser`,
+      providesTags: (result) =>
+        result ? result.map(({ id }) => ({ type: 'SystemUsers', id })) : ['SystemUsers'],
+    }),
+    getSystemUser: builder.query<SystemUser, string>({
+      query: (id) => `systemuser/${id}`,
```

Comment:
Query path parameters should be normalized. This will fail if `id` contains a `/` or a `?` for instance.